hack/jenkins/e2e.sh

```diff
@@ -224,6 +224,7 @@ E2E_OPT=${E2E_OPT:-""}
 # Set environment variables shared for all of the GCE Jenkins projects.
 if [[ ${JOB_NAME} =~ ^kubernetes-.*-gce ]]; then
   KUBERNETES_PROVIDER="gce"
+  : ${GCE_SERVICE_ACCOUNT:=$(cat /var/lib/jenkins/.gce/service_account)}
```

Comment:
this file doesn't exist on all of our VMs. (In particular, none of the PR VMs have it.)

I wonder if using `gcloud auth` would be better?